### PR TITLE
900: Add field nuernberg pass id

### PR DESCRIPTION
**Pull request by sarahsporck**
Waiting for PDF.

### CODE REVIEW

File: specs/card.proto [hunk 1]
```diff
@@ -20,6 +20,10 @@ message NuernbergPassNumberExtension {
   optional uint32 pass_number = 1;
 }
 
+message NuernbergIdExtension {
+  optional uint64 pass_id = 1;
+}
```
Comment (michael-markl) [inline specs/card.proto:25]: Drive-By Comment:

AFAIK, we should not add this to the CardInfo: It should only be printed onto the PDF.
The only use case for it is, to apply for the discount of the Deutschlandticket; embedding this into all QR codes just bloats the QR codes imo.

File: administration/src/cards/extensions/nuernbergPassIdExtension.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  setProtobufData: (state, message) => {
+    message.extensionNuernbergPassId = {
+      passId: BigInt(state.nuernbergPassId),
+    }
+  },
+  causesInfiniteLifetime: () => false,
+  createForm: function (state, setState) {
+    return (
+      <FormGroup
+        label='Pass-Id'
```
Comment (f1sh1918) [inline administration/src/cards/extensions/nuernbergPassIdExtension.tsx:19]: should be renamed to "Nürnberg-Pass-ID"

File: administration/src/cards/extensions/nuernbergPassIdExtension.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        labelFor='nuernberg-pass-input'
+        intent={this.isValid(state) ? undefined : Intent.DANGER}>
+        <InputGroup
+          id='nuernberg-pass-input'
```
Comment (f1sh1918) [inline administration/src/cards/extensions/nuernbergPassIdExtension.tsx:23]:
```suggestion
          id='nuernberg-pass-id-input'
```
should be unique

File: administration/src/cards/extensions/nuernbergPassIdExtension.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    message.extensionNuernbergPassId = {
+      passId: BigInt(state.nuernbergPassId),
+    }
+  },
+  causesInfiniteLifetime: () => false,
+  createForm: function (state, setState) {
+    return (
+      <FormGroup
+        label='Pass-Id'
+        labelFor='nuernberg-pass-input'
```
Comment (f1sh1918) [inline administration/src/cards/extensions/nuernbergPassIdExtension.tsx:20]:
```suggestion
        labelFor='nuernberg-pass-id-input'
```
should be unique

File: administration/src/cards/extensions/nuernbergPassNumberExtension.tsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+type NuernbergPassNumberState = { passNumber: number }
+
+const nuernbergPassNumberLength = 8
+const nuernberg_pass_number_extension: Extension<NuernbergPassNumberState, null> = {
+  getInitialState: () => null,
+  createForm: function (state, setState) {
+    return (
+      <FormGroup
+        label='Passnummer'
+        labelFor='nuernberg-pass-input'
```
Comment (f1sh1918) [inline administration/src/cards/extensions/nuernbergPassNumberExtension.tsx:14]:
```suggestion
        labelFor='nuernberg-pass-number-input'
```
should be unique

File: administration/src/cards/extensions/nuernbergPassNumberExtension.tsx [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+type NuernbergPassNumberState = { passNumber: number }
+
+const nuernbergPassNumberLength = 8
+const nuernberg_pass_number_extension: Extension<NuernbergPassNumberState, null> = {
+  getInitialState: () => null,
+  createForm: function (state, setState) {
+    return (
+      <FormGroup
+        label='Passnummer'
```
Comment (f1sh1918) [inline administration/src/cards/extensions/nuernbergPassNumberExtension.tsx:13]:
```suggestion
        label='Nürnberg-Pass-Nummer'
```

File: administration/src/cards/extensions/nuernbergPassIdExtension.tsx (hunk 2 above)
Comment (f1sh1918) [inline administration/src/cards/extensions/nuernbergPassIdExtension.tsx:19]:
```suggestion
        label='Nürnberg-Pass-ID'
```

**Comment by sarahsporck** [comment]
I opened a new pr as there have been a lot of changes since I last worked on this. I also tried to apply all comments also in the new pr #977, too.
